docs/l2-l3.rst

```diff
@@ -243,3 +243,45 @@ Delete Self IPs
 2. When prompted, click Confirm.
 
 The self IPs are deleted from the BIG-IP device, as well as from the application database.
+
+-------
+
+FAQ
+===
+
+**Q. Why do I get an error for VLAN/self IP delete operation from the App?**
+
+This is a known issue for BIG-IP v 12.x. If a pool with nodes is associated with a self IP of the same subnet, BIG-IP doesn’t allow user to delete that self IP. As a result, the VLAN delete operation also fails with the error.
+
+**Workaround:**
+
+- Delete the corresponding pool member from BIG-IP.
+
+- Perform the VLAN/self IP delete from App.
+
+- Recreate the pool member on BIG-IP.
+
+------
+
+**Q. When I try to stitch a VLAN tag, why do I see “VLAN not available for stitching”? Not able to configure the VLAN.**
```

Comment:
not sure what the message is (is it just "VLAN not available for stitching"?)  if so, it should be something like:
...why do I see "VLAN not available for stitching", and I am unable to configure the VLAN?